temporalcli/commandsgen/commands.yml

```diff
@@ -917,34 +924,32 @@ commands:
   - name: temporal operator nexus
     summary: Commands for managing Nexus resources (EXPERIMENTAL)
     description: |
-      Nexus commands enable managing Nexus resources.
+      These commands manage Nexus resources.
 
       Nexus commands follow this syntax:
 
       ```
-      temporal operator nexus [command] [command] [command options]
+      temporal operator nexus [command] [subcommand] [options]
       ```
 
   - name: temporal operator nexus endpoint
     summary: Commands for managing Nexus Endpoints (EXPERIMENTAL)
     description: |
-      Endpoint commands enable managing Nexus Endpoints.
+      These commands manage Nexus Endpoints.
 
-      Endpoint commands follow this syntax:
+      Nexus Endpoint commands follow this syntax:
 
       ```
-      temporal operator nexus endpoint [command] [command options]
+      temporal operator nexus endpoint [command] [options]
       ```
 
   - name: temporal operator nexus endpoint create
-    summary: Create a new Nexus Endpoint (EXPERIMENTAL)
+    summary: Create a Nexus Endpoint (EXPERIMENTAL)
     description: |
-      Create a new Nexus Endpoint on the Server.
+      Create a Nexus Endpoint on the Server.
 
-      An endpoint name is used in workflow code to invoke Nexus operations.  The
-      endpoint target may either be a worker, in which case `--target-namespace` and
-      `--target-task-queue` must both be provided, or an external URL, in which case
-      `--target-url` must be provided.
+      A Nexus Endpoint name is used in Workflow code to invoke Nexus Operations.
+      The endpoint target may either be a Worker, in which case `--target-namespace` and `--target-task-queue` must both be provided, or an external URL, in which case `--target-url` must be provided.
```

Comment:
Can you wrap lines at whatever the convention is? (120 characters?)

Reply:
Maybe there's a smart way to have the docs gen code to do that for the emitted mdx files, but it seemed out of scope for this set of changes, so this seemed like a reasonable compromise.

Reply:
Hi @prasek and @josh-berry 

This is why we put items on separate lines, nominally sentences:

- Improves version control diffs for easier line-by-line inspection.
- Focuses feedback and review discussions on single points, preventing overlap with other content that may need changes.
- Reduces overlap and merge conflicts with other tickets.
- Enhances the precision of our automated lint tooling, such as Vale (`vale /path/to/file.mdx`), allowing you to more easily locate and address issues by line number.

I don't think that will be a problem here, even with the generated material splitting a sentence.

Reply:
@josh-berry are you good leaving it as-is single sentence per line or do we go back to 80 chars/line?

Reply:
reverted to 80 chars